css/default.css

```diff
@@ -92,3 +92,10 @@ td.print-overdue    { font-weight: bold; }
 .table-nonfluid {
    width: auto !important;
 }
+
+/* Markdown */
+.markdown-body blockquote {
```

Comment:
I don't see why we would require a .markdown-body.  There will be a lot of places where markdown will be rendered due to calls to strings_display function calls that are not just issue details.  For example, the news deprecated function, or other areas.